app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const { getTopics } = require('./controllers/topics.controllers');
+
+const app = express();
+
+app.use(express.json());
+
+app.use((err, req, res, next) => {
+    console.log(err);
+});
```

Comment:
This is not actually doing anything and is also not tested for so should not be here either